src/services/LoggingService.js

```diff
@@ -12,20 +12,24 @@ class LoggingService {
 
   static logError(error) {
     if (typeof newrelic !== 'undefined') {
+      // Note: customProperties are not sent.  Presumably High-Security Mode is being used.
       newrelic.noticeError(error);
     }
   }
 
+  // Note: will simply log errors that don't seem to be API error responses.
   static logAPIErrorResponse(error) {
-    let { message } = error;
+    let processedError = Object.create(error);
     if (error.response) {
-      message = `${error.response.status} ${error.response.config.url} ${JSON.stringify(error.response.data)}`;
+      processedError = new Error(`API request failed: ${error.response.status} ${error.response.config.url} ${JSON.stringify(error.response.data)}`);
     } else if (error.request) {
-      message = `${error.request.status} ${error.request.responseURL} ${error.request.responseText}`;
-    } else if (error.stack) {
-      message = error.stack;
```

Comment:
FYI: Errors with stack are now passed through, instead of creating a new Error, so I _think_ this should be ok.

Reply:
Should LoggingService be pulled out into a separate library so we can all share this code rather than having different copies with different implementations?

Reply:
@georgebabey: As we pull this and some other utilities into the Cookie Cutter, we'll have to decide whether to create a package or not.  There are some trade-offs either way, but I lean toward packages.  Maybe there is a decision tree we could slowly put together to help us decide?  This is a good fedx topic both generally, as well as more specifically regarding logging, analytics, i18n, other?